src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt

```diff
@@ -3176,15 +3176,15 @@ class CourtCasesToNomisIntTest : SqsIntegrationTestBase() {
           )
 
           courtSentencingMappingApi.stubGetAppearanceRecallMappings(
-            "dc71f3c5-70d4-4faf-a4a5-ff9662d5f714",
```

Comment:
The major change in the NOMIS api, it needs to current court events to delete them